### DISCUSSION TITLE
Custom error message

### DISCUSSION THREAD

**Discussion by khanhpd2002** [9tigerio/db2rest#585] [General]
Hi @kdhrubo , I have a question is how can I custom error message.
It seems like db2rest just catch exception from database then show that error like 

`ERROR: null value in column \"notification\" of relation \"notification_configs\" violates not-null constraint\n  Detail: Failing row contains (24, null, thanh, Đây là thông báo test\nCảm ơn! abc, https://upload.aggregatoricapaci.com/green-cross-guild-dev/2024/..., 2024-05-21 03:52:55.670022, 2024-05-21 03:52:55.670022).`

Can we custom it to like "Can null value in column `notification`"

Thank you.

**Comment by ntananh** [comment]
Hi @khanhpd2002 , currently we not support for cusome error message.

**Comment by kdhrubo** [comment]
Are you using docker or jar version of DB2Rest? And which database? @khanhpd2002.
The reason for asking is - we can make it possible to plugin a custom exception translator - which will translate yr database error code , sql state into custom message.
